lib/performance_data/statistics.rb

```diff
@@ -0,0 +1,216 @@
+require 'json'
+require 'gds_api/performance_platform/data_out'
+
+module PerformanceData
+  class Statistics
+    attr_accessor :data_out, :content, :slug, :part_urls, :document_type, :is_multipart, :performance_data
+
+    def initialize(content, slug)
+      @data_out = GdsApi::PerformancePlatform::DataOut.new("http://www.performance.service.gov.uk")
+      @content = content
+      @slug = slug
+      @part_urls = get_part_urls(content, slug)
+      @document_type = @content["document_type"] || @content["format"]
+      @is_multipart = @part_urls.any? || (document_type == 'smart-answer')
```

Comment:
I've also found 2 extra candidates in the `content-schemas` repo as doc types that could be considered multipart:

- TravelAdvice:  https://github.com/alphagov/govuk-content-schemas/blob/ffc129287ae61b1c09190c2b384ece7a0539224c/formats/travel_advice/publisher_v2/examples/travel_advice.json
- Guide:  https://github.com/alphagov/govuk-content-schemas/blob/b88a5292052fc37185712d993f0c5ffee1b9c291/formats/guide/frontend/examples/guide.json

However, looking at the code to get the url parts: 

```
    def get_part_urls(content, slug)
      details = content.fetch("details")
      return [] unless details.key?("parts")

      parts = details.fetch("parts")

      part_urls = parts.map! do |part|
        if part.key?("web_url")
          URI(part["web_url"]).path
        else
          "/#{slug}/#{part['slug']}"
        end
      end
      part_urls.unshift(slug.insert(0, "/"))
    end
```
Those two candidates don't contain "web_url" as a key. They do contain the structure: 
"details": {
   "parts": {}
}
but we also expect a "web_url" inside the `parts` hash.